vaadin-radio-button-flow-parent/vaadin-radio-button-flow/src/main/java/com/vaadin/flow/component/radiobutton/RadioButtonGroup.java

```diff
@@ -524,6 +528,33 @@ public String getLabel() {
         return getElement().getProperty("label");
     }
 
+    /**
+     * Sets the selection preservation strategy on data change. The default is
+     * {@link SelectionPreservationStrategy#PRESERVE_EXISTING}.
```

Comment:
For `RadioButtonGroup`, the default seems to be `DISCARD`.

Reply:
Fixed the JavaDoc.